### PR TITLE
Add Traidio

### PR DESCRIPTION
**Pull request by Enrique123D**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/traidio/background.png
Comment (DoJapHD) [inline minecraft_servers/traidio/background.png:1]: Compress

File: minecraft_servers/traidio/icon.png
Comment (DoJapHD) [inline minecraft_servers/traidio/icon.png:1]: Are you sure that you want to use this icon?
It's really pixelated

File: minecraft_servers/traidio/icon@2x.png
Comment (DoJapHD) [inline minecraft_servers/traidio/icon@2x.png:1]: Are you sure that you want to use this icon?
It's really pixelated

File: minecraft_servers/traidio/manifest.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  "direct_ip": "traidio.net",
+  "server_wildcards": [
+    "%.traidio.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://panel.traidio.net/overview",
```
Comment (DoJapHD) [inline minecraft_servers/traidio/manifest.json:13]: If website is locked behind Discord authentication, remove

File: minecraft_servers/traidio/manifest.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{
+  "server_name": "traidio",
+  "nice_name": "TraidioNET",
+  "direct_ip": "traidio.net",
+  "server_wildcards": [
+    "%.traidio.net"
```
Comment (DoJapHD) [inline minecraft_servers/traidio/manifest.json:6]: Not a real wildcard
[https://i.imgur.com/hH25GpU.png](https://i.imgur.com/hH25GpU.png)

File: minecraft_servers/traidio/icon@2x.png
Comment (Enrique123D) [reply minecraft_servers/traidio/icon@2x.png:1]: I create a new image and upload it tomorrow

File: minecraft_servers/traidio/manifest.json (hunk 2 above)
Comment (Enrique123D) [reply minecraft_servers/traidio/manifest.json:6]: dns

Comment (DoJapHD) [reply minecraft_servers/traidio/manifest.json:6]: > A wildcard DNS record is a record in a DNS zone that will match requests for non-existent domain names.
As there's no redirect of my non-existent domain, it's not a wildcard.

File: minecraft_servers/traidio/manifest.json (hunk 1 above)
Comment (maaaathis) [reply minecraft_servers/traidio/manifest.json:13]: I think you meant `https://traidio.net` as your website?

**Comment by Enrique123D** [comment]
I have edited the files again today. I hope it's okay now.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**